rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -16,9 +16,9 @@ toc::[]
 [id="rosa-q4-2023_{context}"]
 === Q4 2023
 
-* **Configure custom security groups.** With the release of ROSA CLI (`rosa`) version 1.2.31, administrators can use the `rosa create cluster` and `rosa create machinepool` commands to configure a new cluster or a new machine pool with up to 5 additional custom security groups. Additionally, administrators can create a new machine pool with up to 5 additional custom security groups by using OpenShift Cluster Manager. Configuring custom security groups gives administrators greater control over resource access in new clusters and machine pools. For more information, see xref:../rosa_install_access_delete_clusters/rosa_getting_started_iam/rosa-aws-prereqs.adoc#rosa-security-groups_prerequisites[Security groups].
+* **ROSA CLI update.** The ROSA CLI (`rosa`) was updated to a new version. For information about what has changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.31[ROSA CLI release notes]. For more information about the ROSA CLI (`rosa`), see xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc#rosa-about_rosa-getting-started-cli[About the ROSA CLI].
```

Comment:
Do we want the version number here, also? 
```suggestion
* **ROSA CLI update.** The ROSA CLI (`rosa`) was updated to version 1.2.31. For information about what has changed in this release, see the link:https://github.com/openshift/rosa/releases/tag/v1.2.31[ROSA CLI release notes]. For more information about the ROSA CLI (`rosa`), see xref:../cli_reference/rosa_cli/rosa-get-started-cli.adoc#rosa-about_rosa-getting-started-cli[About the ROSA CLI].
```

Reply:
I do not, that keeps the text templated and keeps the maintenance down. It was very deliberate to not have the version number.